extension/src/prototypes/view/containers/Environments.tsx

```diff
@@ -48,12 +71,59 @@ const ambientOcclusionPropsAtom = atom((get): AmbientOcclusion => {
 export const Environments: FC = () => {
   const environmentType = useAtomValue(shareableEnvironmentTypeAtom);
   const colorMode = useAtomValue(colorModeAtom);
+  const showMapLabel = useAtomValue(showMapLabelAtom);
   const debugSphericalHarmonics = useAtomValue(debugSphericalHarmonicsAtom);
   const shadowProps = useAtomValue(shadowMapPropsAtom);
   const ambientOcclusionProps = useAtomValue(ambientOcclusionPropsAtom);
   const graphicsQuality = useAtomValue(shareableGraphicsQualityAtom) || undefined;
   const antialias = graphicsQuality === "ultra" ? "extreme" : graphicsQuality;
 
+  const styleOverrides: Record<ColorMode, any> = useMemo(
```

Comment:
I think this should be a constant variable, because it doesn't depend on a variable which is used in the component.